watertap/unit_models/uv_aop.py

```diff
@@ -279,6 +278,20 @@ def build(self):
             units=units_meta("time"),
             doc="Exposure time of UV light.",
         )
+        self.reactor_volume = Var(
+            initialize=1,
+            bounds=(1e-18, 10000),
```

Comment:
Can you change all these small lower bounds (1e-18) to 0 and ensure solver convergence is still achieved?

Reply:
Yeah, I've revised all these bounds to `(0,None) `and all tests are passed.